src/Documentation/glossary.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  name: 'Glossary',
+  desc:
+    'This guide is aimed to familiarize the users with definitions to ' +
+    'relevant DVC concepts and terminologies which are frequently used.',
+  contents: [
+    {
+      name: 'Workspace directory',
+      match: ['workspace'],
+      desc:
+        'Also abbreviated as workspace, it is the root directory of a ' +
+        'project where DVC is initialized by running `dvc init` command. ' +
+        'Therefore, this directory will contain a `.dvc` directory as well.'
+    },
+    {
+      name: 'Cache directory',
+      match: ['cache'],
+      desc:
+        'DVC cache is a hidden storage which is found at `.dvc/cache`. This ' +
```

Comment:
it's usually `.dvc/cache` - not always

Reply:
Use `name: 'DVC Cache`, and you can probably copy a `desc` from https://dvc.org/doc/user-guide/dvc-files-and-directories and/or https://dvc.org/doc/commands-reference/cache

> BTW we'll have to review if we need to unify al these definitions throughout docs... But that's for a future issue i.e. #461

Reply:
please address this comment